app/views/admin.py

```diff
@@ -322,32 +322,50 @@ def reports(page):
     Reported = User.alias()
     posts_q = SubPostReport.select(
         Value('post').alias('type'),
+        SubPostReport.id,
         SubPostReport.pid,
         Value(None).alias('cid'),
         User.name.alias('reporter'),
         Reported.name.alias('reported'),
         SubPostReport.datetime,
         SubPostReport.reason,
+        SubPostReport.open,
         Sub.name.alias('sub')
-    ).join(User, on=User.uid == SubPostReport.uid) \
+    ).where((SubPostReport.reason == 'tos') | (SubPostReport.reason == 'spam')) \
+        .join(User, on=User.uid == SubPostReport.uid) \
         .switch(SubPostReport).join(SubPost).join(Sub) \
         .join(Reported, on=Reported.uid == SubPost.uid)
 
+    open_posts_q = posts_q.where(SubPostReport.open == True)
+    closed_posts_q = posts_q.where(SubPostReport.open == False)
+
     comments_q = SubPostCommentReport.select(
         Value('comment').alias('type'),
+        SubPostCommentReport.id,
         SubPostComment.pid,
         SubPostCommentReport.cid,
         User.name.alias('reporter'),
         Reported.name.alias('reported'),
         SubPostCommentReport.datetime,
         SubPostCommentReport.reason,
+        SubPostCommentReport.open,
         Sub.name.alias('sub')
-    ).join(User, on=User.uid == SubPostCommentReport.uid) \
+    ).where((SubPostCommentReport.reason == 'tos') | (SubPostCommentReport.reason == 'spam')) \
```

Comment:
I think it would be better to have a boolean flag in `SubPostReport` and `SubPostCommentReport` to indicate the report should go  to the admins or not. As it is right now the free-text reports are not going to the admins despite not being under the "Sub rule violation" option. This would also allow us to have configurable site-wide report options